src/tools/non-promotion-reasons.md

```diff
@@ -343,15 +346,16 @@ f(C c) {
 Assign the getter to a local variable:
 
 {:.good}
+<?code-excerpt "non_promotion/lib/non_promotion.dart (not-field)" replace="/final.*/[!$&!]/g"?>
 ```dart
 import 'dart:math';
-
-abstract class C {
+// ···
```

Comment:
Not huge fan of how this is automatically added anywhere the `docregion`s end/restart in the excerpts file

Reply:
You can disable this by adding `<?code-excerpt plaster="none"?>` to the top of the file or in the excerpt you don't want to have it:

```
<?code-excerpt "file.dart (region)" plaster="none"?>
```